__tests__/data/deleteUsers/ga/http_response.json

```diff
@@ -0,0 +1,138 @@
+[
+  [
+    {
+      "type": "post",
+      "reqParams": [
```

Comment:
Is there a reason why this is an array and not an object?
That'd give more flexibility in terms of defining this JSON.

Reply:
Also, it improves the readability of the data.

Reply:
In the line `expect(mockInstance).toHaveBeenCalledWith(...resp.reqParams);` we are destructuring the array while we are checking if the `axios` instance with relevant arguments. Through this we don't have to worry about how many we are checking with for what type of `axios` instance & all.

I agree about the readability part. But the thing is that the array type implementation makes it easy for us during validation.